.github/workflows/test_template.yml

```diff
@@ -96,6 +96,16 @@ jobs:
         python-version: ${{ matrix.python-version }}
         channels: defaults, conda-forge
         use-only-tar-bz2: true
+    - name: Install HDF5 and pytables on macOS
+      if: runner.os == 'macOS'
```

Comment:
we should test if we are in optional dependency also. We do not want to install pytables all the times

Reply:
Could you please use the "add suggestion" button to make a suggested change here? I don't know how to test for this.

Reply:
```suggestion
      if: ${{ (runner.os == 'macOS')  &&  (env.EXTRA_DEPENDS != '')  &&  (env.INSTALL_TYPE == 'pip') }}
```